### PR TITLE
DC-438 Use only routes that have slash ending

### PR DESCRIPTION
**Pull request by alvinypyim**
# Why?

So that we won't have duplicate entries in our SEO analysis.

Please refer to the story for the reviewers who have access to our issue tracking system.

# Testing

I will be testing this with a Craft project of ours.

### CODE REVIEW

File: src/public/.htaccess [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+  RewriteCond %{REQUEST_FILENAME} -f [OR]
+  RewriteCond %{REQUEST_FILENAME} -d [OR]
+  RewriteCond %{REQUEST_URI} ^/(favicon\.ico|apple-touch-icon.*\.png)$ [NC]
+  RewriteRule .* - [L]
+
+  RewriteCond %{REQUEST_URI} [^/]$
+  RewriteRule .* %{REQUEST_URI}/ [R=301,L]
+
+  RewriteRule (.+) index.php?p=$1 [QSA,L]
```
Comment (alvinypyim) [inline src/public/.htaccess]: This is based on the one provided by Craft.

**Review by alvinypyim — COMMENTED** [review]
Just to provide more information

**Review by mnoack — APPROVED** [review]

**Comment by alvinypyim** [comment]
Do not merge this as I'm still testing this 🙏.

**Comment by alvinypyim** [comment]
🎉 I've tested this with most of ours sites. 👍

**Review by seanchambo — APPROVED** [review]

**Pull request merged**